ppl/cmd/zar/README.md

```diff
@@ -32,26 +32,24 @@ ln -s zq-sample-data/zng
 
 ## ingesting the data
 
-Let's take those logs and ingest them into a directory.   We'll make it
-easier to run all the commands by setting an environment variable pointing
-to the root of the logs tree.
+Let's take those logs and ingest them into a directory.  Often we'd keep our
+archive somewhere like [Amazon S3](https://aws.amazon.com/s3/), but since we're
+just doing a quick test we'll use local temp space.  We'll make it easier to
+run all the commands by setting an environment variable pointing to the root of
+the logs tree.
 ```
-mkdir ./logs
-set ZAR_ROOT=`pwd`/logs
+mkdir /tmp/logs
+ZAR_ROOT=/tmp/logs
```

Comment:
Since the test data is small, I went ahead and advocated building the archive in `/tmp` (and cited S3 as a logical alternative so readers don't get the mistaken impression that we actually advocate putting stuff in temp storage in production). In the past when we had the user just creating it in the current directory, a bunch of the sample outputs further down in the README had to have `/path/to/ZAR_ROOT` in them, whereas using a known dir like `/tmp` lets us avoid that. That addresses two little things:

1.  Whenever I'd have to update the example outputs when we change functionality (it's happened more than you might think :smile: ), I'd have to keep search & replacing the pathnames from the outputs on my system to put back the `/path/to/ZAR_ROOT` part. Not a crisis, just annoying.  
     
2.  If I can get this PR merged, I'd like to see about writing a ztest that executes as much as possible of the stuff in this README so we don't keep regressing. I expect the output checks in that will be much easier to implement/understand if the pathname is "locked in".